README.md

```diff
@@ -1,9 +1,6 @@
----
-status: draft
----
-# The Commonhaus Foundation (CF)
+# <img src="https://www.commonhaus.org/brand/svg/CF_icon_default.svg" height="32px" /> Commonhaus Foundation
 
-The Commonhaus Foundation champions open-source innovation, focusing on the long-term growth and stability of GitHub-style open-source projects. By fostering a dynamic and inclusive community, we empower contributors to drive the evolution of vital open-source libraries and frameworks through collaborative innovation.
+The Commonhaus Foundation (CF) champions open-source innovation, focusing on the long-term growth and stability of GitHub-style open-source projects. By fostering a dynamic and inclusive community, we empower contributors to drive the evolution of vital open-source libraries and frameworks through collaborative innovation.
```

Comment:
What does "GitHub-style" mean? Is it brand specific? I beat others will ask these questions and wonder if CF projects are bound to GitHub in some way or another. What does that say for projects using other code hosting solutions?

If "GitHub-style" is used as a codeword for a concept might be worth referring to the concept instead of using a branded word. Think tissue paper vs. kleenex, or Coca-Cola for any fizzy drink in Georgia, Atlanta 😉

Reply:
This was a term borrowed from CNCF TOC and probably OpenJS. Feel free to recommend a replacement.

The general gist is: some foundations were formed pre-Git (either GitHub or GitLab). Either platform (and advent of Git in general) have changed how OSS communities operate.

but yes.. that's why "-style" and not just GitHub alone.

Reply:
Not really sure I understand the distinction you're trying to make here Erin. 

> There is a different way of growing and managing communities now that needs less hand-holding than was necessary in the 200x's

And if I did understand, I'm not sure I would even agree with this statement.

Whatever, I don't see how simply removing "GitHub-style" from that sentence would significantly change its meaning (to anyone other than you, perhaps).

Reply:
I don't have an issue with adding or removing. Just asking for an alternate suggestion.

The Apache Way was born with Apache in 1999. Eclipse grew out of IBM in 2001. All this is saying (and again, I adopted words here), is that a lot has changed since then.

These words have been in the README for a long time. Please do suggest alternate wording.